### PR TITLE
tartube: Add version 2.1.070

### LINKED ISSUE #4684: [Request] Add Tartube
Hello,

Could you please consider adding Tartube?

https://github.com/axcore/tartube

Latest release in Github releases page:

https://github.com/axcore/tartube/releases

Thanks

### PR DESCRIPTION
**Pull request by ghost-420**
- Closes #4684

### CODE REVIEW

File: bucket/tartube.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            "bin": "tartube_portable_64bit.bat",
+            "shortcuts": [
+                [
+                    "tartube_portable_64bit.bat",
+                    "Tartube",
+                    "",
+                    "tartube_icon.ico"
+                ]
+            ],
+            "persist": [
```
Comment (Ash258) [inline bucket/tartube.json:19]: Persist is not allowed in architecture

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — CHANGES_REQUESTED** [review]
- Investigate how to conifgure data dir automatically into persist folder

image

image

**Comment by ghost-420** [comment]
* Tartube's default data folder setting does not reside where the Scoop installation does; below screenshot is how you can retrieve the default data folder currently.
python3_HgERgVbM34
python3_V53kAre7FY
I 'scoop uninstall -p tartube', reinstalled tartube, then moved the directory from E:\Videos\Tartube to E:\Tartube
python3_9BJRHW38Bx
OKKmZYSaFd
D873JIZHt6
* youtube-dl is already persisted.
FLXvLHSkZe

**Comment by Ash258** [comment]
> Tartube's default data folder setting does not reside where the Scoop installation does; below screenshot is how you can retrieve the default data folder currently.

... It is configurable

image


> youtube-dl is already persisted.

Why you cannot use system-wide youtube-dl instead of some random mingw version?

**Comment by axcore** [comment]
Hi, I'm the Axmud dev.

@Ash258 asks:

>Why you cannot use system-wide youtube-dl instead of some random mingw version?

At the moment, Tartube does not offer many choices for installation on MS WIndows. The reason is this: I am assuming that everyone is using the MS WIndows installer I created.

If there is another installation method (using Scoop), then I can add support for that. Just tell me what changes you need. For example this:

`"data_dir": "C:\\Users\\vagrant\\scoop\\persist\\tartube\\cosi`

All of that could be added to the code, then future releases of Tartube will work with Scoop, without creating any more work for you.

**Comment by Aankhen** [comment]
I’m not sure I understand the issue, but I wanted to chime in to say it would be great if we took up @axcore’s offer to work together to make Tartube more Scoop-friendly, with a view towards improving and merging this PR. I’m sure I’m not the only one who’d love to see it in Scoop, and it would be a shame to let all this work go to waste.

**Comment by rashil2000** [comment]
Thanks for the reminder @Aankhen 

@axcore sorry for the delay here, but did you get around to configuring the portable version to use the same data_dir as the application itself?

**Comment by axcore** [comment]
> Thanks for the reminder @Aankhen
> 
> @axcore sorry for the delay here, but did you get around to configuring the portable version to use the same data_dir as the application itself?

I am happy to help you with any changes you need.

But, remember: I have never used Scoop and know nothing about it. Is there a reason why the user should not choose their own data directory? 

(Tartube's configuration file already lives in the installation folder, by the way.)

**Comment by rashil2000** [comment]
Scoop aims to be portable, self-contained installer and for portable apps it's much preferred to have all the user files in the installation directory itself (at least as a default).

**Comment by pratikpc** [comment]
Closing as ghost-420:tartube has been deleted.

@Aankhen that is indeed a great idea and I would recommend you to work on a new PR instead so we can add tartube to Scoop quickly.

**Pull request closed without merging**